docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.9"
+
+services:
+  backend:
+    build: .
+    depends_on:
+      db:
+        condition: service_healthy
+    ports:
+      - 8080:8080
+    environment:
+      - CB_HOST
+      - CB_USER
+      - CB_PSWD
+    command: $CB_HOST $CB_USER $CB_PSWD
+    volumes:
+      - .:/app
+    container_name: try-cb-api
+  
+  frontend:
+      build: "https://github.com/couchbaselabs/try-cb-frontend-v2#master"
```

Comment:
I think this will become the 7.0 branch rather than master?
Just doing some tidying up of frontend and will merge soon btw.